templates/templates/header.html

```diff
@@ -17,11 +17,24 @@
           <a href="#main-content">Jump to main content</a>
         </span>
         <ul class="p-navigation__links" role="menu">
-          <li class="p-navigation__link{% if level_1 == 'internet-of-things' %} is-selected{% endif %}" role="menuitem"><a href="/internet-of-things">物联网</a></li>
-          <li class="p-navigation__link{% if level_1 == 'desktop' %} is-selected{% endif %}" role="menuitem"><a href="/desktop">桌面系统</a></li>
-          <li class="p-navigation__link{% if level_1 == 'cloud' %} is-selected{% endif %}" role="menuitem"><a href="/cloud">云</a></li>
-          <li class="p-navigation__link{% if level_1 == 'server' %} is-selected{% endif %}" role="menuitem"><a href="/server">服务器</a></li>
-          <li class="p-navigation__link{% if level_1 == 'containers' %} is-selected{% endif %}" role="menuitem"><a href="/containers">容器</a></li>
+          {% with '/internet-of-things' as url %}
```

Comment:
What do you think about creating a map and iterating through it to generate the link list?

Reply:
Hm, since this is probably more likely changed by frontenders I think it might be good to keep this very transparent.
Lets ask @nottrobin @tbille

Reply:
I very much agree with @b-m-f - keeping this nav in markup is much more straightforward & user-friendly.

I'm not even sure it's worth making a variable out of the URL just do you can use it twice. I would personally just go for:

``` html
<li class="p-navigation__link{% if request.path == '/internet-of-things' %} is-selected{% endif %}" role="menuitem"><a href="/internet-of-things">物联网</a></li>
```

But I don't feel very strongly so which you prefer @b-m-f is fine.